ansible/vars/topo_t0-116.yml

```diff
@@ -164,7 +164,7 @@ configuration_properties:
     swrole: leaf
     spine_asn: 65534
     leaf_asn_start: 4200064600
-    tor_asn_start: 4200065100
+    tor_asn_start: 4200065500
```

Comment:
Can you elaborate the reason for making this change?

Changing asn_start will cause deployed DUTs needs to be redeployed.

Reply:
I updated this value according to hardcoded ASN values in tests/common/plugins/fib.py for topo t0-116
if topology == 't0-116': leaf_asn_start = 4200064600 tor_asn_start = 4200065500
And in tests/common/plugins/fib.py added approach to get ASN values from the topology file if not exist will be used default